.github/workflows/check-urls.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v30.0.0
+        uses: tj-actions/changed-files@v32.1.2
```

Comment:
You can alternatively remove this change and update the PR

Reply:
> "Looks like you have taken a wrong branch again. I would suggest you delete the fork, create a fresh fork and make your change on top of it in the main branch and raise a PR."
This is the main branch. I will only delete my fork if other reviewers on this repo recommend ths. 

> "You can alternatively remove this change and update the PR." 
Checking to see if there's a way to revert this. It's noted this change from a script from other contributors here.

Reply:
Resolved #8748